### PR TITLE
Add Faq54

### PR DESCRIPTION
**Pull request by licaon-kter**

### CODE REVIEW

File: FAQ.md [hunk 1]
```diff
@@ -505,6 +505,31 @@ You can enable/disable lockdown mode in the main menu, using a widget or using a
...
+<a name="FAQ54"></a>
+**(54) How to tunnel all TCP connections through the Tor network if you bought the SOCKS feature?**
+
+First, install [Orbot](market://details?id=org.torproject.android), the Android client for Tor,
+run it, press _Start_, while it connects open its _Settings_ and make sure it's setup to auto-start
+on device start.
+
+In NetGuard's _Advanced_ settings enable _Subnet routing_.
+
+Having purchased the NetGuard SOCKS feature (this is required), just enable it in _Advanced_
```
Comment (M66B) [inline FAQ.md]: SOCKS5 is free, but only available in the GitHub version of NetGuard.

**Comment by M66B** [comment]
Thanks for this pull request.

File: FAQ.md (hunk 1 above)
Comment (licaon-kter) [inline FAQ.md]: Oh, saw it with a shopping basket at one time, indeed I did not see any info on this, so I still believed it was a $$$ feature.

**Comment by licaon-kter** [comment]
Better?

**Comment by M66B** [comment]
Good, thanks.

**Pull request merged**